roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -34,6 +34,33 @@
     mode: 0755
   changed_when: False
 
+- set_fact:
+    pvc_prefix: "{{ openshift_logging_es_pvc_prefix }}"
+
+- set_fact:
+    openshift_logging_pvc_name: "{{ openshift_logging_elasticsearch_pvc_name }}"
```

Comment:
@ewolinetz Do we need to be careful here?  If this role is run the same time as the other logging roles, do we not run the risk of overriding dependent vars?  Or is that not a concern in context of running individual rols?